Components/Avatar/avatar.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="avatar.css">
+    <link rel="stylesheet" href="/index.css">
+    <link rel="stylesheet" href="/css/header.css">
+    <link rel="stylesheet" href="/css/sidebar.css">
+    <link rel="stylesheet" href="/css/components.css">
```

Comment:
Instead of adding each and every style sheet in the head, you can create one styles.css and import all your stylesheets there and then you can link that one stylesheet in the head.

Reply:
Thanks for this helpful suggestion, will follow this 👍🏻